benchmark/stats/curve.go

```diff
@@ -0,0 +1,178 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package stats
+
+import (
+	"crypto/sha256"
+	"encoding/binary"
+	"encoding/csv"
+	"encoding/hex"
+	"fmt"
+	"math"
+	"math/rand"
```

Comment:
Please use internal/grpcrand instead.

Reply:
How are grpcrand's concurrency semantics useful here? Or are we interested in using grpcrand's random seed for testing reasons? Using a global lock seems wasteful on the surface, so I'm just trying to understand why it'd be useful here.

Reply:
It's enforced by `vet.sh` that `math/rand` may not be imported (that's why travis is red).  I'd rather update this to use `grpcrand` than add an exception in vet.

Reply:
Nevermind, we already have exceptions to the rule.  Just add a `\|benchmark` to the `grep -v` in `vet.sh` and you should be fine.

Reply:
Done.